contracts/rounds/RoundsManager.sol

```diff
@@ -14,18 +14,25 @@ contract RoundsManager is ManagerProxyTarget, IRoundsManager {
     // Round length in blocks
     uint256 public roundLength;
 
+    // Lock period of a round as a % of round length
```

Comment:
Let's just add one line comment that says something like: "Transcoders can not change their rates within the `roundLockAmount` % of the end of a round to prevent a last minute change after delegators have bonded."